classes/PodsAPI.php

```diff
@@ -4201,6 +4201,18 @@ public function duplicate_pod_item ( $params ) {
         );
 
         foreach ( $fields as $field ) {
+	        if ( in_array( $field['name'], array(
+		        'ID',
+		        'post_name',
+		        'post_date',
```

Comment:
Also I think we should probably do some `$pod->pod_data['type']` checks here, as well as generally skipping the field `if ( $pod->pod_data['field_id'] == $field['name'] )`.